src/Login.js

```diff
@@ -31,18 +12,29 @@ const Login = () => {
         magicLinkUrl: "http://localhost:9000/authenticate",
         expirationMinutes: 30,
       },
-    };
-    stytch.mountLoginOrCreateUser({
-      elementId: "#magic-link",
-      style: style,
-      config: loginOrCreateUserConfig,
-    });
-  }, []);
+    },
+    style: {
+      fontFamily: '"Helvetica New", Helvetica, sans-serif',
+      button: { backgroundColor: "#106ee9" },
+      input: { textColor: "#090909" },
+    },
+    // Fill this public token in from your Stytch dashboard.
+    publicToken: "public-token-1111-1111-1111",
```

Comment:
Can you make a const `STYTCH_PUBLIC_TOKEN` above stytchProps and then use the variable here? then they can search STYTCH_PUBLIC_TOKEN to replace it

Reply:
https://github.com/stytchauth/stytchjs-react-express/pull/7/commits/ab286c81b6c5a72976a4c42c73ff3b49fa6e2077